db-wrapper/src/main/kotlin/backend/db/bots/BotsDatabase.kt

```diff
@@ -128,10 +108,11 @@ class BotsDatabase {
                     ops.add(
```

Comment:
getOperations fails on empty result, add more checks (such as in getBotsByStrategy) in this and other methods

Reply:
Yeap, this is known issue, will fix this